lib/computer_player.ex

```diff
@@ -1,5 +1,12 @@
 defmodule TicTacToe.ComputerPlayer do
-  def move(board_manager, board) do
-    board |> board_manager.available_positions |> Enum.random
+  @ai Application.get_env(:tic_tac_toe, :ai)
+
+  def move(deps, board, player) do
+    get_ai_move(deps, board, player)
+  end
+  
+  def get_ai_move(deps, board, player) do
```

Comment:
Is this only public so you can test it?

Reply:
tbh I only just started putting more thought into which functions I was making private. But it does break the tests when private.

Reply:
it seems like the tests should test move not get_ai_move

Reply:
I see. Now I'm thinking that since all move does is call `get_ai_move`, perhaps I should just delete `get_ai_move` altogether and move the logic to `move`